examples/c/ex_backup_block.c

```diff
@@ -268,8 +268,8 @@ take_full_backup(WT_SESSION *session, int i)
     } else
         hdir = home_incr;
     if (i == 0) {
-        (void)snprintf(
-          buf, sizeof(buf), "incremental=(granularity=1M,enabled=true,this_id=ID%d)", i);
+        (void)snprintf(buf, sizeof(buf),
+          "incremental=(granularity=1M,enabled=true,this_id=\"ID%d:%d.%d\")", i, i, i);
```

Comment:
I think this code represents a valid test for the problem, but I wonder if the special cases might be better handled in the test suite.  I think `test_backup11.py` has some error testing for incremental backup, so trying out special characters there might be better?

Reply:
Good idea. I'll make that fix.

Reply:
Fixed.